src/main/java/org/opensearch/ad/util/ClientUtil.java

```diff
@@ -1,332 +0,0 @@
-/*
```

Comment:
Maybe add in the PR header if this would have any effects? In general less code is always better.

Reply:
The code in question is not utilized within the current system, thus it can be safely removed without impacting any existing functionalities.